homeworks/rostyslav.havuka_RostyslavHavuka/13-ng-forms/forms/src/app/app.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent implements OnInit {
+  form: FormGroup;
+
+  constructor(private readonly formBuilder: FormBuilder) {
+    this.form = this.formBuilder.group({
+      email: ["", [Validators.required, Validators.email]],
+      password: ["", [Validators.required, Validators.minLength(6)]],
+      checkboxRemember: false,
+    });
+  }
+
+  ngOnInit() {
+    if (localStorage.length != 0) {
+      this.form.patchValue({
+        email: atob(localStorage.getItem("email")),
+        password: atob(localStorage.getItem("password"))
+      })
+    } else {
+      this.form.patchValue({
+        email: "",
+        password: ""
+      })
```

Comment:
You don't need to patch the empty value if it already set above. So please, remove the whole else block